src/main/java/http/HttpRequest.java

```diff
@@ -0,0 +1,40 @@
+package http;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import util.Parser;
+import util.URL;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class HttpRequest {
+    private static final Logger logger = LoggerFactory.getLogger(HttpRequest.class);
+    private String startRequest;
+    private Map<String, String> requestHeader = new HashMap<>();
+
+    public HttpRequest(BufferedReader br) throws IOException {
+        init(br);
+    }
+
+    private void init(BufferedReader br) throws IOException {
+        startRequest = br.readLine();
+        String tempRequest = br.readLine();
```

Comment:
tempRequest같은 변수명은 자제해 주세요.

```suggestion
        String startLine = br.readLine();
```

Reply:
앗 아래 코드를 읽어보니 애매하군요. 그냥 line 정도가 적당하겠어요.